cmd/azure.go

```diff
@@ -29,6 +29,7 @@ import (
 	"net/url"
 	"strconv"
```

Comment:
ab suggested that `mc admin restart` can persist metadata on disk and load it again - after the release.
also have a expiry go-routine to purge old map entries - after the release.